pkg/pool-utils/contracts/RecoveryMode.sol

```diff
@@ -41,11 +41,11 @@ import "./BasePoolAuthorization.sol";
  * In particular, stable pools do not attempt to compute the invariant (which is a complex, iterative calculation
  * that can fail in extreme circumstances), and no protocol fees are collected.
  *
- * In some pools, such as those with Oracles or Price Rates, Recovery mode takes advantage of mathematical properties
- * to safely compute the invariant for the special case of proportional withdrawals, without running the usual more
- * complex code necessary in general (e.g., for non-proportional joins/exits and swaps). This might be enough to
- * support functions like `getRate`, allowing dependent pools to function normally. (Otherwise, the dependent pools
- * can themselves be placed into Recovery Mode.)
+ * In some pools, such as those with Price Rates, Recovery mode takes advantage of mathematical properties to safely
+ * compute the invariant for the special case of proportional withdrawals, without running the usual more complex code
+ * necessary in general (e.g., for non-proportional joins/exits and swaps). This might be enough to support functions
+ * like `getRate`, allowing dependent pools to function normally. (Otherwise, the dependent pools can themselves be
+ * placed into Recovery Mode.)
  *
```

Comment:
The blurb update is alright, but now that I look at this paraghraph it seems like we should remove it? After all, we don't try to compute `getRate` differently or anything like that.

Reply:
Yes; this is mostly left over from when we had a more expansive scope for Recovery Mode, and is probably overkill now.